src/NewsService.Business/GetNewsCommand.cs

```diff
@@ -15,29 +28,116 @@ public class GetNewsCommand : IGetNewsCommand
     private readonly INewsRepository _repository;
     private readonly INewsResponseMapper _mapper;
     private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly IRequestClient<IGetUsersDataRequest> _rcGetUsers;
+    private readonly IRequestClient<IGetDepartmentsRequest> _rcGetDepartments;
+    private readonly IDepartmentInfoMapper _departmentInfoMapper;
+    private readonly ILogger<GetNewsCommand> _logger;
+
+    private UserData GetAuthor(Guid userId, List<string> errors)
+    {
+      try
+      {
+        IOperationResult<IGetUsersDataResponse> response = _rcGetUsers.GetResponse<IOperationResult<IGetUsersDataResponse>>(
+          IGetUsersDataRequest.CreateObj(new List<Guid> { userId })).Result.Message;
+
+        if (response.IsSuccess)
+        {
+          return response.Body.UsersData.FirstOrDefault();
+        }
+
+        _logger.LogWarning(
```

Comment:
_logger.LogWarning(Can not get author. Reason:{errors}, string.Join('\n', response.Errors)));